web/worker_service/style.css

```
@@ -1,3 +1,29 @@
+/* Reuses the TwakeInter faces the Flutter app already ships, instead of pulling
+   Inter from Google Fonts: no cross-origin request, and no extra bytes since the
+   app downloads these same files anyway. The banner is display:none until the
+   mobile check passes, so a desktop visit never fetches them.
+
+   Paths point into the linagora_design_flutter package assets. If that package
+   moves its fonts, the banner silently falls back to Helvetica. */
+@font-face {
+  font-family: "TwakeInter";
```

Stylelint (17.14.0)

[error] Expected no quotes around "TwakeInter" (font-family-name-quotes)

(font-family-name-quotes)